pilot/PilotUserService.swift

```diff
@@ -36,14 +36,17 @@ class PilotUserService: NSObject {
                       completion: @escaping (PilotUser) -> Void,
                       failure: @escaping (HTTPStatusCode) -> Void) {
         
+        let basicCredentials = "\(PilotConfiguration.Thunder.userKey):\(PilotConfiguration.Thunder.userSecret)"
+            .data(using: String.Encoding.utf8)!.base64EncodedString(options: [])
+        
         // Build the authorization headers for the request
         let headers: [String: String] = ["Authorization": "Basic \(basicCredentials)",
             "password": "\(password)"]
         
-        // Build the parameters for the request
-        let parameters: [String: String] = ["email": email]
+        // Parameters
+        let parameters = ["email": email]
         
-        Alamofire.request(endpoint, parameters: parameters, headers: headers)
+        Alamofire.request(ThunderRouter.fetch(nil).url, method: ThunderRouter.fetch(nil).method, parameters: parameters, encoding: ThunderRouter.fetch(nil).encoding, headers: headers)
```

Comment:
It's ok to do but It's going to be change in a new branch acknowledging the redundancy in PilotUserService